components/src/core/score-card/scorecard.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'pxb-scorecard',
+    template: `
+        <mat-card class="root">
+            <div class="header" [style.backgroundColor]="headerColor" [style.color]="headerFontColor">
+                <div class="header-overlay"></div>
```

Comment:
can/should we expose backgroundImage as a prop for convenience so a user doesn't have to make a bunch of CSS rules for all of their cards?

Reply:
I don't think we should expose backgroundImage as a prop.  What if the user wants to change the opacity, size, etc.  I'll argue that most style props should be handled by our classes.